content/en/training/_index.html

```diff
@@ -0,0 +1,106 @@
+---
+title: Training
+bigheader: Kubernetes Training and Certification
+abstract: Training programs, certifications, and partners.
+class: gridPage
+cid: training
+---
+
+<section id="training">
+
+  <div class="main-section" style="padding-top: 100px">
+    <div style="width: 66%; margin-left: 16.5%">
+      <center>
+        <h1>Build your cloud native career</h1>
+        <p>Kubernetes is at the core of the cloud native movement. Training and certifications from the Linux Foundation and our training partners lets you invest in your career, learn Kubernetes, and make your cloud native projects successful.</p>
+      </center>
+    </div>
+  </div>
+  <div class="main-section" style="padding-top: 100px">
+    <div>
+      <h2>Take a free course on edX</h2>
+    </div>
+  <div class="col-container">
+    <div class="col-nav">
+      <center>
+        <h5>
+          <b>Introduction to Kubernetes <br> &nbsp;</b>
+        </h5>
+        <p>Want to learn Kubernetes? Get an in-depth primer on this powerful system for managing containerized applications.</p>
+        <br><br>
+        <a href="https://www.edx.org/course/introduction-to-kubernetes" target="_blank" class="button">Go to Course</a>
+      </center>
+    </div>
+    <div class="col-nav">
+      <center>
+        <h5>
+          <b>Introduction to Cloud Infrastructure Technologies</b>
+        </h5>
+        <p>Learn the fundamentals of building and managing cloud technologies directly from The Linux Foundation, the leader in open source.</p>
+        <br>
+        <a href="https://www.edx.org/course/introduction-to-cloud-infrastructure-technologies" target="_blank" class="button">Go to Course</a>
+      </center>
+    </div>
+    <div class="col-nav">
+      <center>
+        <h5>
+          <b>Fundamentals of Containers, Kubernetes, and Red Hat OpenShift</b>
+        </h5>
+        <p>Learn practical techniques for creating and deploying containers on a Kubernetes cluster using Red Hat® OpenShift Container Platform.</p>
+        <br>
+        <a href="https://www.edx.org/course/fundamentals-of-containers-kubernetes-and-red-hat" target="_blank" class="button">Go to Course</a>
+      </center>
+    </div>
+  </div>
+
+  <div class="main-section" style="padding-top: 100px">
+    <div style="width: 66%; margin-left: 16.5%">
+      <center>
+        <h2>Learn with the Linux Foundation</h2>
+        <p>The Linux Foundation offers instructor-led and self-paced courses for all aspects of the Kuberenetes application development and operations lifecycle.</p>
```

Comment:
Hello @celestehorgan ,

A couple of first impressions 😄,

I have not looked at the layout and CSS in detail, but think that the page should be polished up before setting it loose to the public.

If you have a link for `Training` at the top, you should probably have a link on the bottom of the page as well.
Should the `Training` link in the top nav bar appear before the blog? I would think that anything related to partners and marketing belongs further along the list.

nit: Line60, spelling: Kubernetes

nit: The link that sends you to the training partners is somewhat weird. The page your are sent to is the `partners` page, though you only see the view for training and it makes it seem as if this part of the partners page belongs with the `Training` page.

Reply:
Thank you for your review! @kbhawkey 

To address a couple of your comments:
> Should the Training link in the top nav bar appear before the blog? I would think that anything related to partners and marketing belongs further along the list.

This depends on how you look at the idea of "training" – is training more about accessing knowledge/information about Kubernetes in another format? If so, then it belongs nearer Documentation. If you view "training" as a marketing offering, then it belongs near-ish Community on the top nav.

There's no right answer to this, which is why I'm _really_ hoping someone higher up in the Kubernetes SIG Docs community with more experience than I have can weigh in with a definitive opinion.

For me personally, I took the approach that Training was really an extension of/compliment to Documentation, so I moved it beside Docs in the top nav. Additionally, training/certification is a high-value item for many developer users – if a large corporation decides to start a k8s project, they either prioritize hiring certified developers _or_ they send their existing dev/DevOps teams off for formal training and certification. This also reflects a bit of lived experience/personal bias, too: in the corporate world, Tech Writing (Docs) and Tech Training are often in the same department and report to the same manager :)  Again – happy to hear alternatives. I'm not totally sold on its current placement either.

> nit: The link that sends you to the training partners is somewhat weird. The page your are sent to is the partners page, though you only see the view for training and it makes it seem as if this part of the partners page belongs with the Training page.

[I thought about this too](https://github.com/kubernetes/website/pull/19214#discussion_r382171247). I'm not sure if it makes sense to move the KTP section to this page _exclusively_ however – they are, after all, still 'partners'. Again, comment from someone higher up in the SIG Docs world than I would be very much appreciated.

Reply:
Just catching up--@kbhawkey I had the same feedback about link order. 👍

Reply:
@celestehorgan Thanks for moving "Training" to the right of "Blog", that's a solid choice. 👍